### PR TITLE
add datepicker view

### PR DESCRIPTION
**Pull request by BaniChan**

### CODE REVIEW

File: AltSwiftUI/AltSwiftUI/Source/UIKitViews/UIKitControls.swift [hunk 1]
```diff
@@ -1,6 +1,6 @@
 //
 //  UIKitControls.swift
-//  AltSwiftUI
+//  RakuSwiftUI
```
Comment (nodehi) [inline AltSwiftUI/AltSwiftUI/Source/UIKitViews/UIKitControls.swift]: Project name should be "AltSwiftUI"

File: AltSwiftUI/AltSwiftUI/Source/Views/Controls/DatePicker.swift [hunk 2]
```diff
@@ -0,0 +1,89 @@
+//
+//  DatePicker.swift
+//  RakutenTravelCore-c960a1ef
```
Comment (nodehi) [inline AltSwiftUI/AltSwiftUI/Source/Views/Controls/DatePicker.swift]: Project name should be "AltSwiftUI"

File: AltSwiftUI/AltSwiftUI/Source/UIKitViews/UIKitControls.swift (hunk 1 above)
Comment (BaniChan) [reply AltSwiftUI/AltSwiftUI/Source/UIKitViews/UIKitControls.swift]: updated

File: AltSwiftUI/AltSwiftUI/Source/Views/Controls/DatePicker.swift (hunk 2 above)
Comment (BaniChan) [reply AltSwiftUI/AltSwiftUI/Source/Views/Controls/DatePicker.swift]: updated

**Review by nodehi — APPROVED** [review]

File: AltSwiftUI/AltSwiftUI/Source/Views/Controls/DatePicker.swift [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+public struct DatePicker: View {
+    public var viewStore = ViewValues()
+
+    var selection: Binding<Date>?
+    var minimumDate = Date()
+    var maximumDate: Date?
+    var components: Components
+    
+    public enum DatePickerComponent {
+        case date, hourAndMinute
```
Comment (kevinwl02) [inline AltSwiftUI/AltSwiftUI/Source/Views/Controls/DatePicker.swift]: Please add documentation to each enum case

File: AltSwiftUI/AltSwiftUI/Source/Views/Controls/DatePicker.swift (hunk 2 above)
Comment (wimbledons) [reply AltSwiftUI/AltSwiftUI/Source/Views/Controls/DatePicker.swift]: Looks good!

File: AltSwiftUI/AltSwiftUI/Source/UIKitViews/UIKitControls.swift (hunk 1 above)
Comment (wimbledons) [reply AltSwiftUI/AltSwiftUI/Source/UIKitViews/UIKitControls.swift]: We gonna get altswiftui.com domain.

**Review by kevinwl02 — CHANGES_REQUESTED** [review]
Please see comment regarding documentation

File: AltSwiftUI/AltSwiftUI/Source/Views/Controls/DatePicker.swift (hunk 3 above)
Comment (BaniChan) [reply AltSwiftUI/AltSwiftUI/Source/Views/Controls/DatePicker.swift]: updated

**Review by kevinwl02 — APPROVED** [review]

**Pull request merged**